webinar.html

```diff
@@ -0,0 +1,231 @@
+---
+layout: page
+title: Webinar 
+permalink: /webinar/
+---
+<style>
+    .taglist
+    {
+        display: none;
+        margin-left: 50px;
+    }
+    input[type=checkbox] {
+        float: left;
+        margin-top: 1.4em;
+        margin-right: 0.5em;
+    }
+    input[type=checkbox]:checked ~ ul {
+        display: block;
+    }
+</style>
+
+<div class="entry-content">
+						<p>Every week, we get to watch 30 to 45 minute webinars on various
+ machine learning topics.  These are the webinars that we
+ watched over the last couple of years.</p>
+ 
+ <ul>
+ 
+<li><strong>2021/03/12-5</strong>: <a href="https://www.youtube.com/watch?v=DhHyshhc1lY">Quantization of Neural Networks – High Accuracy at Low Precision</a> </li>
```

Comment:
If we want to embed the youtube video itself in the page, it would be a relatively simple modification, like in the example below:
```suggestion
<li><strong>2021/03/12-5</strong>: Quantization of Neural Networks – High Accuracy at Low Precision </li>
<iframe width="560" height="315" src="https://www.youtube.com/embed/DhHyshhc1lY" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
IMO, embedded videos make for a more attractive page than a list of hyperlinks.

Reply:
Did it for youtube videos.  had problems with facebook videos so I left it as is.